e2e_tests/tests/per_provider/key_mappings.rs

```diff
@@ -27,6 +27,7 @@ fn use_and_check() -> Result<()> {
     client.set_default_auth(Some(String::from("parsec-tool")));
 
     let keys = client.list_keys()?;
+    assert!(!keys.is_empty());
```

Comment:
This might fail for providers that don't have persistent keys installed, like the CAL provider.

Reply:
Yea, I didn't see CAL provider in action. Also the motive for this change is test will pass even for the providers that should have persistent keys but they don't exist.
So do you have something in mind to avoid this false positive?

Reply:
Hi @ionut-arm, sorry I'll be active on this PR again. So any suggestion to avoid false positives?
And by `I didn't see CAL provider in action`, it's vague, I admit. I meant I didn't see keys generated for the CAL provider.
I wonder if we don't run the key_mapping test for the CAL provider until the TODO about the CAL provider in the generate_keys script is done.

Reply:
Yeah, the CAL provider is in somewhat of a limbo, I think it's ok to not run the tests for now. Perhaps add a comment about it in [this](https://github.com/parallaxsecond/parsec/issues/585) issue.

Reply:
Done!